packages/traceloop-sdk/src/lib/tracing/ai-sdk-transformations.ts

```diff
@@ -318,27 +318,49 @@ const transformPrompts = (attributes: Record<string, any>): void => {
 };
 
 const transformPromptTokens = (attributes: Record<string, any>): void => {
-  if (AI_USAGE_PROMPT_TOKENS in attributes) {
-    attributes[`${SpanAttributes.LLM_USAGE_PROMPT_TOKENS}`] =
-      attributes[AI_USAGE_PROMPT_TOKENS];
-    delete attributes[AI_USAGE_PROMPT_TOKENS];
+  if (SpanAttributes.LLM_USAGE_INPUT_TOKENS in attributes) {
+    if (AI_USAGE_PROMPT_TOKENS in attributes) {
```

Comment:
This is where the token data is - if you deleted it before setting the input tokens attribute you lose it forever